x-pack/plugins/uptime/public/lib/helper/rtl_helpers.tsx

```diff
@@ -88,6 +89,7 @@ export function MockRouter<ExtraCore>({
     </Router>
   );
 }
+configure({ testIdAttribute: 'data-test-subj' });
```

Comment:
I've been using `data-test-subj` with rtl for a while now, and now I'm confused and have no idea how it was working. Is this already set at a higher level somewhere in Kibana?

Reply:
ohh really , it could be set at higher level. I wasn't sure about that,